TestFoundation/TestURL.swift

```diff
@@ -619,6 +671,15 @@ class TestURL : XCTestCase {
             ("test_URLResourceValues", testExpectedToFail(test_URLResourceValues,
                 "test_URLResourceValues: Except for .nameKey, we have no testable attributes that work in the environment Swift CI uses, for now. SR-XXXX")),
         ]
+
+#if os(Windows)
+        tests.append(contentsOf: [
+            ("test_pathSeparator", test_pathSeparator),
```

Comment:
Can we add 'Windows' in the test names so it's obvious above who they're for? And/or wrap the tests above in `#if os(Windows)` too, since they don't run anywhere else.

Reply:
yes & yes